README.md

```diff
@@ -27,6 +27,20 @@ documentation at https://articlequality.readthedocs.io .
                      'ga': 0.21617801281707663,
                      'start': 0.13906813268582238}}
 
+## Install
+
+### Requirements
+
+* Python 3.5, 3.6 or 3.7
+* All the system requirements of [revscoring](https://github.com/wikimedia/revscoring)
+
+### Installation steps
+
+1. clone this repository
+2. install its dependencies `python -m pip -r requirements.txt`
+3. install the package itself `python setup.py install`
```

Comment:
If you run `python setup.py install` it will automatically try to gather the requirements that the pip command above is referencing.

Reply:
Mh, I thought that didn't work when I tried it previously; but I attempted it again just now on a fresh git checkout and it works just fine. Maybe I didn't have the correct python version or a virtual env previously?

Anyway, I will remove the 2. step. Thank you for pointing it out.